server/Arcadia.Assistant.InboxEmail/InboxEmailsNotificator.cs

```diff
@@ -100,15 +107,13 @@ private async Task BroadcastNewEmails()
                     throw new Exception("Loading inbox emails error", error.Exception);
 
                 case GetInboxEmails.Success success:
-                    if (this.lastEmailId != null)
-                    {
-                        var emailsEventBus = new EmailsReceivedEventBus(success.Emails.ToList());
-                        Context.System.EventStream.Publish(emailsEventBus);
-                    }
-
                     this.lastEmailId = success.Emails.Max(e => e.UniqueId);
 
-                    break;
+                    var newEmails = this.lastEmailId == null
```

Comment:
never null?

Reply:
Which one? `this.lastEmailId` will be null until any email is loaded. `newEmails` will never be null. It can be, instead of empty enumerable, it may save some processor ticks :)